pipeline_tools/shared/submission/parse_cromwell_metadata.py

```diff
@@ -39,13 +48,18 @@ def main():
     if pipeline_type == "Optimus":
         ref_fasta_path, pipeline_version = parse_optimus_metadata(metadata)
     elif pipeline_type == "SS2":
-        ref_fasta_path, pipeline_version = parse_SS2_metadata(metadata)
+        ref_fasta_path, pipeline_version, single_sample_pipeline_version = parse_SS2_metadata(metadata)
     else:
         raise RuntimeError('pipeline-type must be Optimus or SS2')
 
     with open('pipeline_version.txt', 'w') as f:
         f.write(pipeline_version)
```

Comment:
```suggestion
        f.write(single_sample_pipeline_version)
```

Reply:
this is actually writing pipeline_version which is either multi_sample or optimus, it always exists. likes 59-61 is where single_sample will get written, and only if it exists (it will not exist for optimus).